pkgs/development/python-modules/git-dummy/default.nix

```
@@ -0,0 +1,44 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  installShellFiles,
+  gitpython,
+  typer,
+  pydantic-settings,
+}:
+
+buildPythonPackage rec {
+  pname = "git-dummy";
+  version = "0.1.2";
+  format = "setuptools";
```

```suggestion
  pyproject = true;
```
please use the new style described in language-frameworks/python.

You probably need to add setuptools to build-system